docs/csharp/programming-guide/linq-query-expressions/toc.md

```diff
@@ -1,19 +0,0 @@
-# [LINQ Query Expressions](index.md)
```

Comment:
you're now getting a warning on the build report because this file was removed. I think you need to remove the reference to this folder from https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/toc.md